src/Powercord/coremods/util.js

```diff
@@ -99,5 +100,33 @@ module.exports = {
     Experiments="${experimentOverrides > 0 ? Object.keys(getExperimentOverrides()).join(', ') : 'n/a'}"
     Labs="${enabledLabs.length ? enabledLabs.map(e => e.name).join(', ') : 'n/a'}"
     Plugins="${plugins.join(', ')}"\`\`\``;
+  },
```

Comment:
This should be in the moduleManager util folder. Also make sure you import everything you need.